recipes/natural_language_processing/chatbot/bootc/Containerfile

```diff
@@ -19,6 +19,11 @@ ARG APP_IMAGE=quay.io/ai-lab/${RECIPE}:latest
 ARG SERVER_IMAGE=quay.io/ai-lab/llamacpp_python:latest
 ARG TARGETARCH
 
+# Include growfs service
+COPY build/usr/lib /usr/lib
+COPY build/usr/libexec /usr/libexec
+RUN chmod +x /usr/libexec/bootc-generic-growpart
```

Comment:
Very minor nit, it is better to just make it executable in git.

Reply:
updated

Reply:
I found I had to add `--chmod=0755` to the `COPY` directive still.